Makefile

```diff
@@ -216,6 +216,7 @@ version-bump-dev:
 	go run cmd/tools/main.go bump --dev
 
 commands-reference:
+	mkdir -p docs/docs/cli
```

Comment:
Should it be `docs/docs/cli`, or we should just change `docs.go` to a different directory, like `docs/cli` directly?

Reply:
TBH idk - since these are temporary they should perhaps go to some `/gen` or `/build` directory? what's the go convention for this?